components/director/internal/domain/runtime/service_test.go

```diff
@@ -192,9 +192,7 @@ func TestService_CreateWithMandatoryLabels(t *testing.T) {
 				return tenantSvc
 			},
 			UIDServiceFn: func() *automock.UidService {
-				svc := &automock.UidService{}
-				svc.On("Generate").Return(runtimeID)
-				return svc
+				return &automock.UidService{}
```

Comment:
You can extract 
`func() *automock.UidService {
				return &automock.UidService{}
}`
in separate function named 'unusedUIDService' for example.